cmd/integration_link.go

```diff
@@ -22,6 +22,8 @@ import (
 )
 
 var (
+	reviewAppsFromForksSecurityWarning = "Only allow automatic review apps deployments from forks if you trust the owners of those forks, as this could lead to security issues"
```

Comment:
Sorry, one final request for changes: addition of a link to the documentation as suggested on slack
```suggestion
	reviewAppsFromForksSecurityWarning = "Only allow automatic review apps deployments from forks if you trust the owners of those forks, as this could lead to security issues. More info here: https://doc.scalingo.com/platform/app/review-apps#addons-collaborators-and-environment-variables"
```

Reply:
Yup, done